frappe/public/js/frappe/form/control.js

```diff
@@ -26,16 +26,27 @@ frappe.ui.form.Control = Class.extend({
 		}
 	},
 	make: function() {
+		this.make_container();
 		this.make_wrapper();
+		this.container.attr("data-fieldtype", this.df.fieldtype);
 		this.$wrapper
 			.attr("data-fieldtype", this.df.fieldtype)
 			.attr("data-fieldname", this.df.fieldname);
 		this.wrapper = this.$wrapper.get(0);
 		this.wrapper.fieldobj = this; // reference for event handlers
 	},
 
+	make_container: function() {
```

Comment:
Adding classes to wrapper was the first attempt; however solving the centering problem with flex box was murky. A control element has two or three children elements, which all have to retain their relative (left) alignment to each other, while being collectively centered in the form. For that, I kept their wrapper as is, and centered it itself inside another container.